cli/command/system/version.go

```diff
@@ -76,14 +76,16 @@ func runVersion(dockerCli *command.DockerCli, opts *versionOptions) error {
 	}
```

Comment:
My thought was to make the ` (downgraded from ...)` part of the template, that way, "presentation" is separate from the "data". Doing so, allows the `--format` tag to use the "raw" version, without having a rather "meaningless" `<version> (downgraded from <other version>)`.